ReadMe.md

```diff
@@ -1,28 +1,36 @@
-BeWake screensaver
-======================
-
-
-Installation
--------------
-
-Simply drag and drop the BeWake file over the link "Drop BeWake here". Open the ScreenSaver Preferences and select BeWake from the listbox.
-
-
-Copyright
-------------
-
-Original 2003 version by Mattia Tristo, enhanced and updated for Haiku at BeGeistert 22 in 2009 by Mattia Tristo and Gabriele Biffi.
-Released under the terms of the GPL License v2.
-
-
-Contact
-----------
-
-Write us at mattia.tristo@gmail.com or gb@biffuz.it
-If you have any suggestion or comment about the code, don't esitate to write us.
-
-
-Special thanks
-------------------
-
-Many thanks to Jon Pimble for his suggestion and to have published the source code of his screensaver Sparticles.
+# BeWake screensaver
+A screensaver for BeOS.
+
+![BeWake screenshot](bewake.png)

```

Comment:
There's lots of black in that shot and the dots are in a bit of a boring arrangement. Can you provide a more interesting shot, maybe scaled down even, to save some space?